Connecting to database specified by database.yml


Started GET "/assets/rails.png" for 10.0.2.2 at 2013-05-28 14:58:50 +0000
Served asset /rails.png - 200 OK (103ms)
Connecting to database specified by database.yml


Started GET "/" for 10.0.2.2 at 2013-05-28 15:00:19 +0000

ActionController::RoutingError (No route matches [GET] "/"):
  actionpack (3.2.13) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  actionpack (3.2.13) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.13) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.13) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.13) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.13) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.13) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.13) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.13) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.13) lib/rails/engine.rb:479:in `call'
  railties (3.2.13) lib/rails/application.rb:223:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.13) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /home/vagrant/.rbenv/versions/1.9.3-p429/lib/ruby/1.9.1/webrick/httpserver.rb:138:in `service'
  /home/vagrant/.rbenv/versions/1.9.3-p429/lib/ruby/1.9.1/webrick/httpserver.rb:94:in `run'
  /home/vagrant/.rbenv/versions/1.9.3-p429/lib/ruby/1.9.1/webrick/server.rb:191:in `block in start_thread'


  Rendered /home/vagrant/.rbenv/versions/1.9.3-p429/lib/ruby/gems/1.9.1/gems/actionpack-3.2.13/lib/action_dispatch/middleware/templates/rescues/routing_error.erb within rescues/layout (6.8ms)
Connecting to database specified by database.yml
Connecting to database specified by database.yml
Connecting to database specified by database.yml
  [1m[36m (0.1ms)[0m  [1mselect sqlite_version(*)[0m
  [1m[35m (7.0ms)[0m  CREATE TABLE "schema_migrations" ("version" varchar(255) NOT NULL) 
  [1m[36m (6.8ms)[0m  [1mCREATE UNIQUE INDEX "unique_schema_migrations" ON "schema_migrations" ("version")[0m
  [1m[35m (1.9ms)[0m  SELECT "schema_migrations"."version" FROM "schema_migrations" 
Migrating to CreateUsers (20130528150144)
  [1m[36m (0.1ms)[0m  [1mbegin transaction[0m
  [1m[35m (4.7ms)[0m  CREATE TABLE "users" ("id" INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, "created_at" datetime NOT NULL, "updated_at" datetime NOT NULL) 
  [1m[36m (1.2ms)[0m  [1mINSERT INTO "schema_migrations" ("version") VALUES ('20130528150144')[0m
  [1m[35m (3.6ms)[0m  commit transaction
  [1m[36m (1.9ms)[0m  [1mSELECT "schema_migrations"."version" FROM "schema_migrations" [0m
Connecting to database specified by database.yml
Connecting to database specified by database.yml
Connecting to database specified by database.yml
Connecting to database specified by database.yml
  [1m[36m (1.7ms)[0m  [1mSELECT "schema_migrations"."version" FROM "schema_migrations" [0m
Migrating to CreateUsers (20130528150731)
  [1m[35m (0.1ms)[0m  select sqlite_version(*)
  [1m[36m (0.0ms)[0m  [1mbegin transaction[0m
  [1m[35m (0.1ms)[0m  CREATE TABLE "users" ("id" INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, "name" varchar(255), "class_year" integer, "netid" varchar(255), "created_at" datetime NOT NULL, "updated_at" datetime NOT NULL) 
SQLite3::SQLException: table "users" already exists: CREATE TABLE "users" ("id" INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, "name" varchar(255), "class_year" integer, "netid" varchar(255), "created_at" datetime NOT NULL, "updated_at" datetime NOT NULL) 
  [1m[36m (0.1ms)[0m  [1mrollback transaction[0m
